### ISSUE TITLE
A11y_ConnectedFieldService-SolutionDeploymentApp_ShareIotalert_ScreenReader: NVDA does not announce the table name.

### ISSUE THREAD

**Issue by 12-shweta** [nvaccess/nvda#10360]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags: #A11yMAS; #A11yTCS; #MAS4.1.2;#MAS4.2.1; #MAS1.3.1 ; #DesktopWeb ;#Chrome; #NVDA

**Environment Details:​**
Application Name: Connected Field Service- Solution Deployment App
User Name – OCETestTenant@OCERegion.onmicrosoft.com
Password – Mic**S***t
Operating System: Win10​
Edge:44.17763.1.0
Chrome Version: Version 76.0.3809.100 (Official Build) (64-bit)
NVDA Version : 2019.2

**Repro Steps:**​
1. Launch the URL https://monti1tiecrmlivesg619org0.crm2.crmlivetie.com 
2.Navigate to site map and Select IoT Alerts.
3.Navigate to any record from the grid and click on it.​
4.Navigate to More commands(...) besides to Add to Queue and click on it.​
5.Navigate to "Share" list item and hit enter.​

6.Start NVDA.
7.Navigate till table using caps + arrows.
8.Observe the announcement by screen reader.

**Actual:​**
Screen reader doesn't announce the table name.
Similar issue is also observed with Narrator.

**Same issue is observed in following screens:**
1.SwitchProcess:1. Launch the application Url and select Connected field service from the Published App list
2.Navigate to site map and Select IoTAlerts.​
3.Navigate to any record from the grid and click on it.​
4.Navigate to "Process" menu item and click on it.​
5.Navigate to Switch Process list item and click on it ​
6. Verify that all the elements of " SwitchProcess" screen are accessible and MAS Compliant.

**Expected:**
Screen reader should announce the table name.

**​User Impact:​**

If screen reader only users wants to navigate only through the table then they get confused on which table they are navigating.

**Recommendations:**​
Please refer to below URL which is repository for accessibility bug fixes.​
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx​

**​MAS Reference:​**

MAS4.1.2:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}​
​
MAS1.3.1:​
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS4.2.1:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

Attachments : 
![1561103_A11y_ConnectedFieldService-SolutionDeploymentApp_ShareIotalert_ScreenReader](https://user-images.githubusercontent.com/54656883/66467523-323d7200-eaa2-11e9-85e7-1a2f91d685d2.png)

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
